File: PRAK204-2310817220016-NailaHanifah.c
#include <stdio.h>
#include <math.h>
#define phi 3.142857
int main() {
    double r, t;
    scanf("%lf %lf", &r, &t);
    double volume = phi * pow(r, 2) * t;
    double luas = 2 * phi * r * (r + t);
    double keliling = 2 * phi * r;
    printf("Volume = %.2lf\n", volume);
    printf("Luas = %.2lf\n", luas);
    printf("Keliling = %.2lf\n", keliling);
    return 0;
}

File: PRAK204-2310817220016-NailaHanifah.py
phi = 22/7
r, t = map(float, input().split())
volume = phi * r * 2 * t
luas = 2 * phi * r * (r + t)
keliling = 2 * phi * r
print(f"Volume = {volume:.2f}")
print(f"Luas = {luas:.2f}")
print(f"Keliling = {keliling:.2f}")
